Flow.Launcher/Helper/DataWebRequestFactory.cs

```diff
@@ -2,68 +2,67 @@
 using System.IO;
 using System.Net;
 
-namespace Flow.Launcher.Helper
+namespace Flow.Launcher.Helper;
+
+public class DataWebRequestFactory : IWebRequestCreate
 {
-    public class DataWebRequestFactory : IWebRequestCreate
+    class DataWebRequest : WebRequest
     {
-        class DataWebRequest : WebRequest
-        {
-            private readonly Uri m_uri;
-
-            public DataWebRequest(Uri uri)
-            {
-                m_uri = uri;
-            }
+        private readonly Uri _uri;
```

Comment:
Renamed from `m_uri` to make it more consistent with the rest of the codebase. It's `private` so it should be safe to rename.